src/main/java/javax/validation/constraints/Past.java

```diff
@@ -64,6 +78,11 @@
 	Class<? extends Payload>[] payload() default { };
 
 	/**
+	 * @return true if the present date should be considered valid
+	 */
+	boolean orPresent() default false;
```

Comment:
What is the "granularity" of present? E.g. when validating

    @Past(orPresent=true) Year year = Year.of(2016);

Would that be true for every day in 2016?